src/Tribe/PUE/Checker.php

```diff
@@ -235,7 +235,7 @@ private function set_slug( $slug = '' ) {
 		 * @return string
 		 */
 		public function get_pue_update_url() {
-			$pue_update_url = 'https://pue.tri.be';
+			$pue_update_url = 'https://pue.theeventscalendar.com';
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 4c094b58f6ab557b0bbcd871f1c40006</sub>